packages/core/src/components/button/stories/button.core.stories.tsx

```diff
@@ -44,6 +44,15 @@ export const Secondary: StoryObj = {
   },
 }
 
+export const OutlineFilled: StoryObj = {
+  render: (args) => createButton(args, 'dark'),
+  args: {
+    ...Primary.args,
+    fill: 'outline-filled',
+    color: 'secondary',
+  },
+}
+
 export const White: StoryObj = {
```

Comment:
guess we could change this title (White) to `Ghost` or `Outline`

Reply:
Yes, I think a good idea

Reply:
done at [69c86f1](https://github.com/juntossomosmais/atomium/pull/508/commits/69c86f1895638bb4b455ccfc85bf7a8f5dc7803b)